src/main/java/com/eleks/academy/who_am_i/networking/Server.java

```diff
@@ -0,0 +1,44 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintStream;
+import java.net.ServerSocket;
+import java.net.Socket;
+
+public class Server {
+
+	public static void main(String argvs[]) throws Exception {
+
+		ServerSocket serverSocket = new ServerSocket(888);
+		System.out.println("Server starts");
+		System.out.println("Waiting for a client connect....");
+
+		Socket clientSocket = serverSocket.accept();
+		System.out.println("Connected with client");
+
+		PrintStream ps = new PrintStream(clientSocket.getOutputStream());
+
+		BufferedReader br = new BufferedReader(new InputStreamReader(clientSocket.getInputStream()));
+
+		BufferedReader kb = new BufferedReader(new InputStreamReader(System.in));
+
+		while (true) {

```

Comment:
Why do you need this loop if you perform `System#exit` in the very first iteration?